src/main/resources/templates/home.html

```diff
@@ -8,7 +8,7 @@
     <div th:replace="main.html::phase"></div>
     <div class="grid-row">
         <div class="column-two-thirds">
-            <h1 class="homepage-intro" th:text="${register.text}">The register of UK government registers.</h1>
+            <div class="homepage-intro" th:utext="${@uk.gov.register.presentation.HtmlViewSupport@renderMarkdown(register.text)}">The register of UK government registers.</div>
 
```

Comment:
no. `th:utext` means "unescaped text" which is what we want if we want the tags created by the markdown render to be rendered as tags rather than escaped as entities like `&lt;`.